splunk-otel-android/src/main/java/com/splunk/rum/ConfigFlags.java

```diff
@@ -26,7 +26,8 @@ class ConfigFlags {
     private boolean networkMonitorEnabled = true;
     private boolean anrDetectionEnabled = true;
     private boolean slowRenderingDetectionEnabled = true;
-    private boolean backgroundTaskInstrumentationEnabled = true;
+    private boolean isSubprocessInstrumentationEnabled = true;
+    private boolean isBackgroundInstrumentationDeferredUntilForeground = false;
```

Comment:
So the javabean style getters for these should have the `is` prefix, but the variables themselves should not.
```suggestion
    private boolean subprocessInstrumentationEnabled = true;
    private boolean backgroundInstrumentationDeferredUntilForeground = false;
```